src/components/Sum/Sum.jsx

```diff
@@ -1 +1,15 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    Sum of
+    {' '}
```

Comment:
it's a bad practice to add spaces this way, it's better to use margins or paddings for that 
check other places